sql/migrate/lex.go

```diff
@@ -54,9 +54,40 @@ func Stmts(input string) ([]*Stmt, error) {
 	}).Scan(input)
 }
 
+// FileStmtDecls scans atlas-format file statements using
+// the Driver implementation, if implemented.
+func FileStmtDecls(drv Driver, f File) ([]*Stmt, error) {
```

Comment:
Not sure I like the naming. I see `StmtDecls` is taken, but why `File`? We can extend the `StmtDecls` with a spread and use options like `Driver()` maybe?

Reply:
There's a method under `File` named `StmtDecls`. This is the same as calling File.StmtDecls, but accepts Driver as the first argument. Feels simpler and clear enough to me. Especially if I want to have both Stmts and StmtDecls.